pages/token/[id].tsx

```diff
@@ -185,7 +185,6 @@ const Token: React.FC<Props> = () => {
       ) : (
         '-'
       ),
-      tooltipTitle: token?.supply || '0',
```

Comment:
This field doesn't appear in other diffs, is it just a patch for previous updates?